knimin/static/js/platemapper.js

```diff
@@ -113,11 +116,42 @@ PlateMap.prototype.initialize = function (data) {
       }
     }
 
+    // Create the GUI
     this._drawPlate();
+
+    // If this plate was already partially full, populate the interface with
+    // the information received
+    if (data.layout.length > 0) {
+      this._populatePlate(data.layout);
+    }
 };
 
 /**
  *
+ * Populates the GUI with the plate information
+ *
+ * @param {Array} layout The plate information
+ **/
+PlateMap.prototype._populatePlate = function (layout) {
```

Comment:
Yes! When updating a well, is triggering the even `change` which will then fire off all the validations.